invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/base.js

```diff
@@ -78,8 +78,8 @@ export function DashboardResultView(props) {
         </Grid.Row>
         <Overridable id="DashboardResultView.resultFooter" {...props}>
           <Grid.Row verticalAlign="middle">
-            <Grid.Column width={4} />
-            <Grid.Column width={8} textAlign="center" floated="right">
+            <Grid.Column className="computer tablet only" width={4} />
```

Comment:
same comment as in https://github.com/inveniosoftware/invenio-communities/pull/762 here but if that is not the way then ignore this, otherwise looks good!

Reply:
I think it is a very good point, I would say the invenio-search-ui would be the module to place it in